RecoVertex/BeamSpotProducer/test/BeamFit.cc

```diff
@@ -390,7 +390,7 @@ void fitAll() {
   fit(xmat, Verr);  // get initial values
   fnthite++;
 
-  Double_t chi2cut = 20;
+  //Double_t chi2cut = 20;
```

Comment:
This was only commented out instead of deleted since a few lines below there is commented out code which uses that variable.